pkgs/applications/blockchains/zcash/default.nix

```diff
@@ -49,15 +113,19 @@ rustPlatform.buildRustPackage.override { inherit stdenv; } rec {
     configureFlagsArray+=("RUST_VENDORED_SOURCES=$cargoDepsCopy")
   '';
 
-  CXXFLAGS = [
-    "-I${lib.getDev utf8cpp}/include/utf8cpp"
-    "-I${lib.getDev cxx-rs}/include"
-  ];
+  preConfigure = ''
+    export CFLAGS="-pipe -O3 -Wno-unknown-warning-option"
+    export CXXFLAGS="-pipe -O3 -Wno-unknown-warning-option -I${lib.getDev utf8cpp}/include/utf8cpp -I${lib.getDev cxx-rs}/include"
```

Comment:
Shouldn't this come out of the configure scripts?

Reply:
I ran in to issues:
```
configureFlags = [
...
    "\"CFLAGS=-pipe -O3 -Wno-unknown-warning-option\""
    "\"CXXFLAGS=-pipe -O3 -Wno-unknown-warning-option -I${lib.getDev utf8cpp}/include/utf8cpp -I${lib.getDev cxx-rs}/include\""

];
```

```
       > ./configure
       > ./src/univalue/configure
       > ./src/secp256k1/configure
       > configure flags: --disable-dependency-tracking --prefix=/nix/store/m147r0qyas0w4wqslnvb9rwl1qgcvzxg-zcash-5.6.1 --disable-tests --disable-bench --with-boost-libdir=/nix/store/y7vnhpxnpdj8jjwyv4vvqmv9gir6mdjj-boost-1.82.0/lib RUST_TARGET=x86_64-unknown-linux-gnu \"CFLAGS=-pipe -O3 -Wno-unknown-warning-option\" \"CXXFLAGS=-pipe -O3 -Wno-unknown-warning-option -I/nix/store/qlyjqqsxx86hvc4249c9wd1i0bcmdk00-utf8cpp-3.2.3/include/utf8cpp -I/nix/store/mbk1rdfp04q723h58hlc6qbpgn0d2p1y-cxx-rs-1.0.94-dev/include\" RUST_VENDORED_SOURCES=/tmp/nix-build-zcash-5.6.1.drv-0/cargo-vendor-dir
       > configure: error: invalid variable name: `"CFLAGS'
```

Reply:
Without double quotes:
```
      > configure flags: --disable-dependency-tracking --prefix=/nix/store/jdp007mcjnhvw2izmkbymkp184hmhlj7-zcash-5.6.1 --disable-tests --disable-bench --with-boost-libdir=/nix/store/y7vnhpxnpdj8jjwyv4vvqmv9gir6mdjj-boost-1.82.0/lib RUST_TARGET=x86_64-unknown-linux-gnu CFLAGS=-pipe -O3 -Wno-unknown-warning-option CXXFLAGS=-pipe -O3 -Wno-unknown-warning-option -I/nix/store/qlyjqqsxx86hvc4249c9wd1i0bcmdk00-utf8cpp-3.2.3/include/utf8cpp -I/nix/store/mbk1rdfp04q723h58hlc6qbpgn0d2p1y-cxx-rs-1.0.94-dev/include RUST_VENDORED_SOURCES=/tmp/nix-build-zcash-5.6.1.drv-0/cargo-vendor-dir
       > configure: error: unrecognized option: `-O3'
```

Reply:
We do we even need to provide this manually to the configure script?

Reply:
ugghh, I don't know...